src/transformers/models/encodec/modeling_encodec.py

```diff
@@ -786,7 +786,8 @@ def forward(
         >>> audio_codes = outputs.audio_codes
         >>> audio_values = outputs.audio_values
         ```"""
-        return_dict = return_dict or self.config.return_dict
+        if return_dict is None:
+            return_dict = self.config.return_dict
```

Comment:
Here we could do instead: 

```suggestion
        return_dict = return_dict if return_dict is not None else self.config.return_dict
```
so that `return_dict` is still set to self.config.return_dict by default.

Reply:
It would be also nice to do the same modification in the `decode` method (at this [line](https://github.com/huggingface/transformers/blob/f183580d0541f895d941dd4ce2393c237c25e243/src/transformers/models/encodec/modeling_encodec.py#L732)).